napari/_qt/menus/file_menu.py

```diff
@@ -248,5 +248,5 @@ def show_shortcuts(viewer: 'Viewer'):
     viewer.window.file_menu._open_preferences()
     pref_list = viewer.window.file_menu._pref_dialog._list
     for i in range(pref_list.count()):
-        if pref_list.item(i).text() == "Shortcuts":
+        if pref_list.item(i).text() == trans._("Shortcuts"):
```

Comment:
I believe this is coming from the field title of `NapariSettings.shortcuts`, so should also be translated to work in all languages. @Czaki : I think you added this function in #4628, can you confirm?

Reply:
TBD for latter: if it's supposed to be the exact same word for comparison then I think we should put it in a common file and imports as otherwise there is a risk in translated napari that one of the two is different from the other I belive.

Reply:
And/Or maybe used `deferred=True` and use `trans._('Shortcuts').value()` for logic comparisons like this?

Reply:
Yeah, I think in that case that will work indeed.